src/components/CreateUser/CreateUserForm.js

```diff
@@ -124,11 +124,46 @@ export default class CreateUserForm extends React.Component {
       .auth()
       .createUserWithEmailAndPassword(email, passHash)
       .then(({ user }) => {})
-      .catch(error => {
-        console.log(error);
-        let newErr = error.message || "";
-        newErr = newErr.replace("email address", "username");
-        this.setState({ waiting: false, errorMessage: newErr || "failed to create user" });
+      .catch(err => {
+        console.log(err);
+        let newMsg = err.message;
+        switch (err.code) {
+          case "auth/invalid-email":
+            newMsg =
+              "Invalid username inputted. Usernames must only have alphanumeric characters plus !@#$%.";
```

Comment:
inputted isn't a word apparently, just remove it and have it start with "Invalid username: "